src/assert.libsonnet

```diff
@@ -229,4 +229,575 @@
     name: name,
     value: value,
   }),
+
+  /*
```

Comment:
I have not implemented cases for httpOnly and session, so that they are searched in cookie.extensions[], in case they are present in set-cookie header.

My intent with this comment is to let maintainer of assert functions know why there is no such case in the function, although Cookie.httpOnly and  Cookie.session exist.

What explanation are you thinking about?